### ISSUE TITLE
Usability: print credentials success message

### ISSUE THREAD

**Issue by tjbaker** [cevoaustralia/aws-google-auth#45]
I'd love to see a summary success message like the following:

https://github.com/ThoughtWorksInc/aws_role_credentials/blob/master/aws_role_credentials/actions.py#L27-L33

Related, this project prints an export statement though I wonder how often it is used.  This project encourages the use of the `profile` best practice, so wonder why people need to export the vars.

**Comment by mide** [comment]
That's a very valid question. I think we could add a flag `-e`, `--export` to only put out the export line, and by default print something more human friendly.

That would also make it easier to use this tool with other scripts:
```bash
$(aws-google-auth ...)
```
instead of 
```bash
$(aws-google-auth ... | tail -n 1)
```

**Comment by nonspecialist** [comment]
yes, I added that because my initial workflow was to do something like `eval $(aws-google-auth | grep export)` (not exactly, but you get the idea). Having it an option flag would make this cleaner while retaining the functionality.
